cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/PullCommand.java

```diff
@@ -81,28 +83,35 @@ public void execute() {
 
         String resourceName = argList.get(0);
         String orgName;
+        String packageName;
         String moduleName;
         String version;
 
         // Get org-name
-        int orgNameIndex = resourceName.indexOf("/");
-        if (orgNameIndex != -1) {
-            orgName = resourceName.substring(0, orgNameIndex);
-            if (orgName.equals("ballerina")) {
-                throw LauncherUtils.createLauncherException("`Ballerina` is the builtin organization and its modules"
-                                                                    + " are included in the runtime.");
-            }
-        } else {
-            throw LauncherUtils.createLauncherException("no module-name provided");
+        if (!validateModuleName(resourceName)) {
+            CommandUtil.printError(outStream,
+                    "invalid module name. Provide the module-name with the org-name ",
+                    "ballerina pull {<org-name>/<module-name> | <org-name>/<module-name>:<version>}",
+                    false);
+            Runtime.getRuntime().exit(1);
+            return;
+        }
+
+        String[] moduleInfo = resourceName.split("/");
+        orgName = moduleInfo[0];
+        packageName = moduleInfo[1];
+        if (orgName.equals("ballerina") || orgName.equals("ballerinax")) {
```

Comment:
This should be removed we should not special case ballerina org

Reply:
Removed.